pkg/cli/cmds/k3sopts.go

```diff
@@ -94,3 +83,61 @@ func commandFromK3S(cmd *cli.Command, flagOpts map[string]*K3SFlagOption) (*cli.
 	cmd.Flags = newFlags
 	return cmd, errs.Err()
 }
+
+// flagSetUsage receives a flag and a K3S flag option, parses
+// both and sets the necessary fields based on the underlying
+// flag type.
+func flagSetUsage(flag cli.Flag, opt *K3SFlagOption) {
+	v := reflect.ValueOf(flag).Elem()
+	if v.CanSet() {
+		switch t := flag.(type) {
```

Comment:
Do we need to switch on type if all types support Usage the same way?

Reply:
Easier and cleaner means of underlying type access for assignment purposes without direct reflection invocation.